### ISSUE TITLE
Remove <div> wrapper from <button>

### ISSUE THREAD

**Issue by vdvibhu20** [coding-blocks/codingblocks.online.projectx#413]
There are two primitive button class types used in our platform:-
`.button-solid` & `.button-dashed`

In some files, `<button>` elements are wrapped inside `<div>`. Remove the `<div>` wrapper and apply all the classes and actions to `<button>` respectively. 

For instance, you will find in `/app/pods/components/otp-enroll/template.hbs::line22`
```
<div class="button-solid" {{action 'handleClick'}}>
  <button>{{buttonText}}</button>
</div>
```
After refactoring it should look like
`<button class="button-solid" {{action 'handleClick'}}>{{buttonText}}</button>`

**Comment by sahilbhosale63** [comment]
Hi, I am working on this issue.

**Comment by farhan787** [comment]
@abhishek97 please review :-)

**Comment by Anant016** [comment]
I will send a pull request by today.

**Comment by Anant016** [comment]
> There are two primitive button class types used in our platform:-
> `.button-solid` & `.button-dashed`
> 
> In some files, `<button>` elements are wrapped inside `<div>`. Remove the `<div>` wrapper and apply all the classes and actions to `<button>` respectively.
> 
> For instance, you will find in `/app/pods/components/otp-enroll/template.hbs::line22`
> 
> ```
> <div class="button-solid" {{action 'handleClick'}}>
>   <button>{{buttonText}}</button>
> </div>
> ```
> 
> After refactoring it should look like
> `<button class="button-solid" {{action 'handleClick'}}>{{buttonText}}</button>`

Sir, it's done in each and every file accordingly, Please check.

**Comment by Anant016** [comment]
Would you please acknowledge the work, and merge my pull request.

**Comment by Archaeologist03** [comment]
@vdvibhu20 Hi there, I fixed this one with #476 
Is this good? Those were pretty much all of the button-solid div wrapped buttons, button-dashed were already fixed.

**Comment by Sushant2** [comment]
this issue is still open?
please assign it to me!
